packages/node-fetch/src/fetchNodeHttp.ts

```diff
@@ -109,8 +116,19 @@ export function fetchNodeHttp<TResponseJSON = any, TRequestJSON = any>(
 
         outputStream ||= new PassThrough();
 
-        nodeResponse.pipe(outputStream, {
-          end: true,
+        pipeThrough({
+          src: nodeResponse,
+          dest: outputStream,
+          signal,
+          onError: e => {
```

Comment:
Based on pipeThrough implementation, it seems that this clean up is already done by the function, why do we need to also implement it there ?